.github/workflows/helm-charts.yaml

```diff
@@ -0,0 +1,35 @@
+name: Release Charts
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Configure Git
+        run: |
+          git config user.name "$GITHUB_ACTOR"
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+
+      - name: Install Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.8.1
+
+      - name: Run chart-releaser
+        uses: helm/chart-releaser-action@v1.4.0
```

Comment:
Are we sure that ASF Infra has whitelisted and approved using this Github Action?

Reply:
Thanks for raising up your concerns. Do you mean that publishing the helm chart to github packages does not conform to the Apache bylaws? I am not an expert in this area :)

BTW, we do not mean to change the official release process. The released helm charts are still hosted in `https://downloads.apache.org/flink/ `.

cc @gyfora

Reply:
I think it has been approved by INFRA recently, see https://issues.apache.org/jira/browse/INFRA-21731 and also https://infra.apache.org/github-actions-policy.html

Reply:
I think this should be fine as it is approved by the apache infra. This is only for convenience of exposing the different helm chart versions on github. As @wangyang0918 said this is not part of the official release process.

Reply:
Since Infra has whitelisted it, there shouldn't be an issue. It does make sense to pin the commit hash of the action (like the policy asks). The concern is not so much related to the release process, but such action could be used for credential stealing etc.

Reply:
If only pushing to `main` branch, not the pull requests, will trigger the chart-releaser-action, do you think we still have the risk of credential stealing?

Reply:
The attack vector is that the workflow could be changed/compromised to perform an unwanted/unverified change, such as outputting credentials to a log file or submitting it to a 3rd party command & control server. So yes, the risk would still be there, only (much) lower. If you pin the workflow with a SHA commit, you would mitigate it even more (because any change in the workflow would only happen in a newly created commit, which you then wouldn't include in your action run)

Reply:
Thanks. I get your point.